test/utils.jl

```diff
@@ -69,6 +65,30 @@ function test_loglik(D::Type{<:Distribution}; atol::Float64 = 1e-3, rtol::Float6
     return
 end
 
+function test_link_unlink(D::Type{<:Distribution})
+    n_params = ScoreDrivenModels.num_params(D)
+    t = 1
+    param_tilde = zeros(1, n_params)
+    param = ones(1, n_params)
+    ScoreDrivenModels.link!(param_tilde, D, param, t)
```

Comment:
I would add a test to verify if the value is the same after linking and unlinking

Reply:
This is the test, if we link and then unlink on top of `param` it should still be a `ones(1, n_params)`

Reply:
Whoops